buildbot/steampipe_app_build.vdf

```diff
@@ -0,0 +1,48 @@
+"AppBuild"
```

Comment:
I was thinking it should be here as a buildbot-related thing since it relies on the folder layout of the extracted 7z + dmg files. (For example, [the `Dolphin-x64` subdir is named by the buildbot during 7z compression](https://github.com/dolphin-emu/sadm/blob/9da3d33905554e3eaa8bbab7c067df2f71827edf/buildbot/master.cfg#L188).)

Reply:
But things like which binaries should/shouldn't be included is really a property of the build itself, no?

Could we somehow make this VDF file more independent of buildbot implementation details if we have the .sh build script do a bit more massaging with the extraction of the archives?

I agree it's kind of borderline though... Not sure one way or another.

Reply:
> Could we somehow make this VDF file more independent of buildbot implementation details if we have the .sh build script do a bit more massaging with the extraction of the archives?

Oh, definitely. Just need to make sure everything is in `win`, `mac`, and `lin` directly and get rid of everything unnecessary that we exclude using `FileExclusions`. Then we can have something like this:

```
{
    "AppID" "1941680"
    "Desc" "DOLPHIN_BUILD_NUMBER"
    "SetLive" "dev-sniper"

    "ContentRoot" "content"
    "BuildOutput" "output"

    "Depots"
    {
        "1941682" // Windows
        {
            "FileMapping"
            {
                "LocalPath" "win/*"
                "DepotPath" "."
                "Recursive" "1"
            }
        }
        "1941683" // macOS
        {
            "FileMapping"
            {
                "LocalPath" "mac/*"
                "DepotPath" "."
                "recursive" "1"
            }
        }
        "1941684" // Linux
        {
            "FileMapping"
            {
                "LocalPath" "lin/*"
                "DepotPath" "."
                "Recursive" "1"
            }
        }
    }
}
```